src/ethereum_test_specs/blockchain.py

```diff
@@ -643,7 +648,8 @@ def make_fixture(
                     t8n, t8n_state=alloc, expected_state=block.expected_post_state
                 )
         self.check_exception_test(exception=invalid_blocks > 0)
-        self.verify_post_state(t8n, t8n_state=alloc)
+        if not self.exclude_post_state:
+            self.verify_post_state(t8n, t8n_state=alloc)
```

Comment:
```suggestion
        self.verify_post_state(t8n, t8n_state=alloc)
```
After thinking about it, maybe this skipping this check is not necessary because, for example, we could have a huge state because of the size of the contracts, but as a test writer I would still like to know that some code ended up in the addresses I expect them to, so I could add a check in the post for the nonces of each contract